llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -233,3 +233,6 @@ def FlattenedThreadIdInGroup : DXILOpMapping<96, flattenedThreadIdInGroup,
                                              "Provides a flattened index for a "
                                              "given thread within a given "
                                              "group (SV_GroupIndex)">;
+def Frac : DXILOpMapping<22, unary, int_dx_frac,
```

Comment:
Would be good to keep the records sorted by opcode number. So, consider moving this to be between `Sin` and `UMax`.

Reply:
that makes more sense I was sorting by name, b\c in hlsl_intrinsics.h it is name.